DBO/SpecialistDbo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ZnajdzLekarza.DBO
+{
+    public class SpecialistDbo
+    {
+        public int ID { get; set; }
+
+        public string FirstName { get; set; }
+
+        public string LastName { get; set; }
+
+        public string Email { get; set; }
+
+        public string PhoneNumber { get; set; }
+
+        public string Description { get; set; }
+
+        public string Education { get; set; }
+
+        public virtual ICollection<LocalizationDbo> LocalizationsDbos { get; set; }
```

Comment:
Powinno być tylko jedno "s" na końcu nazwy. Nawet VS chyba sam tak podpowiada i wtedy baza dobrze sie tworzy. Jeśli "Dbo" zostanie to powinno być LocalizationDbos. Chociaż wydaje mi sie że powinno być po prostu Localizations i obiekt Localization.

Reply:
Masz rację z tym pojedynczym 's'